Sources/StytchCore/StytchB2BClient/StytchB2BClient+Routes.swift

```diff
@@ -15,8 +15,14 @@ extension StytchB2BClient {
         case oauthRoute(OAuthRoute)
 
         var path: Path {
-            let (base, next) = routeComponents
-            return "b2b".appendingPath(base).appendingPath(next.path)
+            switch self {
+            case let .bootstrap(route):
```

Comment:
oof, so all B2B bootstrap requests were getting the b2b prepended and thus never completing with accurate bootstrap data 😬 Glad you caught and fixed it!

Reply:
Yeah, initially I couldn't for the life of me figure out why the calls weren't working.